src/groupeddataframe/fastaggregates.jl

```diff
@@ -238,12 +238,11 @@ end
 
 function (agg::Aggregate{typeof(var)})(incol::AbstractVector, gd::GroupedDataFrame)
     means = groupreduce((x, i) -> x, Base.add_sum, agg.condf, /, false, incol, gd)
+    z = zero(eltype(incol))
```

Comment:
This should be more precise I think:
```suggestion
    z = zero(eltype(incol)) - zero(eltype(means))
```